tests/gateway/test_fluent.py

```diff
@@ -129,3 +131,73 @@ def test_fluent_get_gateway_uri(gateway):
     set_gateway_uri(gateway_uri=gateway.url)
 
     assert get_gateway_uri() == gateway.url
+
+
+def test_fluent_query_chat(gateway):
+    set_gateway_uri(gateway_uri=gateway.url)
+    routes = search_routes()
+    expected_output = {
+        "candidates": [
+            {
+                "message": {
+                    "role": "assistant",
+                    "content": "The core of the sun is estimated to have a temperature of about "
+                    "15 million degrees Celsius (27 million degrees Fahrenheit).",
+                },
+                "metadata": {"finish_reason": "stop"},
+            }
+        ],
+        "metadata": {
+            "input_tokens": 17,
+            "output_tokens": 24,
+            "total_tokens": 41,
+            "model": "gpt-3.5-turbo-0301",
+            "route_type": "llm/v1/chat",
+        },
+    }
+
+    data = {"messages": [{"role": "user", "content": "How hot is the core of the sun?"}]}
+
+    mock_query = mock.Mock(return_value=expected_output)
+
+    with mock.patch(
+        "mlflow.gateway.fluent.MlflowGatewayClient", return_value=mock.Mock(query=mock_query)
```

Comment:
```suggestion
        "mlflow.gateway.fluent.MlflowGatewayClient.query", return_value=mock.Mock(query=mock_query)
```

We can directory mock `query`. That's more straightforward.

Reply:
good idea :)